test/test_torch.py

```diff
@@ -6545,15 +6545,20 @@ def inner(self, device, dtype):
     ('floor_divide', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1, 1e-5, 1e-5, _types),
     ('floor_divide', 'tensor', _small_3d,
         lambda t, d: [_small_3d(t, d, has_zeros=False)], 1, 1e-5, 1e-5, _types),
-    ('pow', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('pow', '1', _small_3d, lambda t, d: [_number(1., 1, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('pow', '2', _small_3d, lambda t, d: [_number(2., 2, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('pow', '3', _small_3d, lambda t, d: [_number(3., 3, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('pow', '-1', _small_3d, lambda t, d: [_number(-1., -1, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
+    ('pow', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1e-1, 1e-1, 1e-5, 
```

Comment:
Instead of updating these tests implement an OpInfo for pow and remove all the entries here.

Reply:
One of these six tests is only supposed to be for non-half types, so I think I shouldn't remove it:
```
    ('pow', '-2', _small_3d, lambda t, d: [_number(-2., -2, t)],
        1e-1, 1e-5, 1e-5, _float_types_no_half, _cpu_types, False),
```

Also, I think I shouldn't remove [most entries](https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_methods_invocations.py#L2486-L2498) for `pow()` from `method_tests()` in `common_methods_invocations`.py, as all of them are implicitly dtype-specific, and seem to supplement these. But I'd remove them if you'd advise otherwise. Thanks!

Reply:
We should remove all of them. The sample_inputs() function each OpInfo has can accept any dtype. sample_inputs() can also inspect the dtype it's generating inputs for to specialize them, so we shouldn't lose any test coverage.